test/common/fixtures/plugins/otel_metrics/kibana.json

```diff
@@ -0,0 +1,15 @@
+{
+  "id": "openTelemetryInstrumentedPlugin",
+  "owner": {
+    "name": "Stack Monitoring",
+    "githubTeam": "stack-monitoring-ui"
+  },
+  "version": "1.0.0",
+  "kibanaVersion": "kibana",
+  "requiredPlugins": [
```

Comment:
I spent *hours* trying to figure out why there was no data coming from prometheus endpoint, despite having all configuration in place. 

Turns out ,`metricsCollection` plugin has to be initialised first, because`@opentelemetry/api-metrics` is a singleton and `setGlobalMeterProvider` (see [here](https://github.com/elastic/kibana/pull/135999/files#diff-cc07ce777d17b9fafe6933322c29cfa1cdefb7f0049d74467113ce620e84e019R127)) must be configured prior to defining new metrics

Reply:
Might be good to put that in the monitoring_collection readme

Reply:
done